feature-libs/product-configurator/rulebased/root/rulebased-configurator-root.module.ts

```diff
@@ -13,10 +13,10 @@ import { VariantConfiguratorOverviewModule } from './variant/variant-configurato
 @NgModule({
   imports: [
     CommonModule,
-    CommonConfiguratorModule,
```

Comment:
This is breaking change. (according to "Removing anything from Angular modules, such as imports, providers, declarations, entryComponents, or exports.")

Reply:
picked

Reply:
Done..